src/main/resources/db/migration/V6__insert_user_role.sql

```diff
@@ -0,0 +1,2 @@
+INSERT INTO user_role(user_id, role_id) VALUES(1, 2);
```

Comment:
here I think need to be VALUES(1, 2) -> VALUES(1, 1), because user_id=1 is admin and role_id=1 also is admin